configs/mocha-config-compass/register/why-node-running.js

```diff
@@ -0,0 +1,12 @@
+require('why-is-node-running/include');
+exports.mochaHooks = {
+  afterAll() {
+    const timeout = setTimeout(() => {
+      console.log(
+        "if the process still running, run kill -SIGINFO %s to see what's keeping it",
+        process.pid
+      );
+    }, 10000);
+    timeout.unref?.();
```

Comment:
This was pretty helpful for debugging tests hanging issue related to react (see above) so I decided to keep it. Tell me if you would prefer that I remove this or put it behind an env variable or something